static/extensions/TheYali1/Alert Extension.js

```diff
@@ -0,0 +1,151 @@
+(function (Scratch) {
+    'use strict';
+  
+    class AlertExtension {
+      constructor() {}
+  
+      showAlert(args) {
+        alert(args.TEXT);
+      }
+  
+      promptText(args) {
+        return prompt(args.MESSAGE || 'Insert text here:');
+      }
+  
+      confirmAction(args) {
+        return confirm(args.MESSAGE || 'are you sure?');
+      }
+  
+      WebExt_DisplayNotification(args) {

```

Comment:
the notifications part of this extension seems a bit pointless since there is already an extension for it on the TurboWarp Gallery, one of our listed galleries in penguinmod